xml_converter/src/attribute/festival_filter.cpp

```diff
@@ -7,9 +7,9 @@
 #include "../string_helper.hpp"
 
 using namespace std;
+//  Functionality will change for parsing
```

Comment:
No nee to comment this out, just delete it. If we need access to what it used to be we can look back into the git history.